installer/templates/new/widgets/text/text.scss

```diff
@@ -19,14 +19,20 @@ $moreinfo-color:    rgba(255, 255, 255, 0.7);
 
   .more-info {
     color: $moreinfo-color;
+    font-size: 24px;
   }
 
   .updated-at {
     color: rgba(255, 255, 255, 0.7);
   }
 
 
-  &.large h3 {
+  & h3 {
```

Comment:
the text widget defined a size for the main text but the `.large` class wasn't actually being added. It was fairly redundant so I simply removed it and now the widget respects the `h3` settings.